frontend/src/pages/NewRunV2.tsx

```diff
@@ -505,26 +508,22 @@ function NewRunV2(props: NewRunV2Props) {
         {/* One-off/Recurring Run Type */}
         {/* TODO(zijianjoy): Support Recurring Run */}
         <div className={commonCss.header}>Run Type</div>
-        {/* <div>Only one-off run is supported for KFPv2 Pipeline at the moment.</div> */}
-        {isClone && <span>{isRecurringRun ? 'Recurring' : 'One-off'}</span>}
-        {!isClone && (
-          <React.Fragment>
-            <FormControlLabel
-              id='oneOffToggle'
-              label='One-off'
-              control={<Radio color='primary' />}
-              onChange={() => setIsRecurringRun(false)}
-              checked={!isRecurringRun}
-            />
-            <FormControlLabel
-              id='recurringToggle'
-              label='Recurring'
-              control={<Radio color='primary' />}
-              onChange={() => setIsRecurringRun(true)}
-              checked={isRecurringRun}
-            />
-          </React.Fragment>
-        )}
+        <React.Fragment>
```

Comment:
NIT: You can change `<React.Fragment>` to `<>` for simplification. Besides that, I am curious if this `<React.Fragment>` can be removed?

Reply:
Yes, it can be removed